app/views/exercises/implement.html.slim

```diff
@@ -6,12 +6,19 @@
 #editor-column
   - unless @embed_options[:hide_exercise_description]
     .exercise.clearfix
-      div
-        span.badge.rounded-pill.bg-primary.float-end.score
+      div.col-3.float-end
```

Comment:
We still have some issues with smaller screens here, and could potentially improve the layout a little. Not too important, but just something I noticed.

<img width="627" alt="Bildschirmfoto 2023-09-03 um 23 03 52" src="https://github.com/openHPI/codeocean/assets/7300329/a637dc7e-dd61-4fc6-bb66-0612e092bdfd">